### ISSUE TITLE
Wrong date on ticket view page

### ISSUE THREAD

**Issue by nandi96** [osTicket/osTicket#6111]
Hi,

I have a problem with the time zone on the custom fields.
When I choose a date (just date, without time), and I save, then shows wrong date with time hour and minute on the ticket view page.
After I refreshing the page, showing correctly.
The problem is visible on the osticketawesome view and the old vanilla view too.

Please view the video about the problem here: https://streamable.com/pnnf6i

What is the problem?

-
![NAS date settings](https://user-images.githubusercontent.com/97993114/159517244-03eba0b5-8bbd-42e3-8229-138e346a954a.png)
![osticket informations](https://user-images.githubusercontent.com/97993114/159517253-2a58d053-e021-4710-b743-0e56d2e49b40.png)
![PHP timezone settings](https://user-images.githubusercontent.com/97993114/159517256-3f2817c6-2371-4a26-bfdc-34dd0eb084e9.png)
![timezone settings](https://user-images.githubusercontent.com/97993114/159517260-a8ba38c6-5be2-4fe0-babf-a44276ddf06c.png)
![User profile timezone settings](https://user-images.githubusercontent.com/97993114/159517261-0d7ea251-d8a2-4a1c-8d10-03ffdf2cd66a.png)

**Comment by JediKev** [comment]
@nandi96 

Yes, the date is returned in UTC instead of in the Agent's/System's timezone on inline edit. We already have this on our bug sheet to look into.

Cheers.

**Comment by nandi96** [comment]
@JediKev any updates maybe?
Thanks, N.
